src/main/kotlin/frc/robot/subsystems/drive/ProfiledPosePID.kt

```diff
@@ -39,32 +40,58 @@ private val rotationalLimits
             rotationalMaxAcceleration.get()
         )
 
-val xController
-    get() =
-        ProfiledPIDController(
-            xGains.kP.get(),
-            xGains.kI.get(),
-            xGains.kD.get(),
-            linearLimits
+var xController =
+    ProfiledPIDController(
+        xGains.kP.get(),
+        xGains.kI.get(),
+        xGains.kD.get(),
+        linearLimits
+    )
+var yController =
+    ProfiledPIDController(
+        yGains.kP.get(),
+        yGains.kI.get(),
+        yGains.kD.get(),
+        linearLimits
+    )
+
+var thetaController =
+    ProfiledPIDController(
+            thetaGains.kP.get(),
+            thetaGains.kI.get(),
+            thetaGains.kD.get(),
+            rotationalLimits
         )
-val yController
-    get() =
-        ProfiledPIDController(
-            yGains.kP.get(),
-            yGains.kI.get(),
-            yGains.kD.get(),
-            linearLimits
+        .apply { enableContinuousInput(-Math.PI, Math.PI) }
+val controllers =
+    mapOf(
+            "x" to { xController },
+            "y" to { yController },
+            "theta" to { thetaController }
```

Comment:
```suggestion
            "ϴ" to { thetaController }
```
This is how AdvantageScope shows theta.
Edit: I think GitHub doesn't render this but it's the actual Greek letter.

Reply:
0697dcef61228af7cc4dc1a590b3431bb7199acd